app/controllers/api/v1/courses_controller.rb

```diff
@@ -38,6 +37,10 @@ def index
 
     private
 
+      def per_page
```

Comment:
How comes we're overriding the `per_page` method here with essentially the same logic as the one defined in https://github.com/DFE-Digital/teacher-training-api/pull/1547/files#diff-100c2681a1f561f361ad2a624c23c8feR22?

Reply:
v1 seems to work differently compared to the other endpoints. If you actually look at all the v1 endpoints they never actually call `paginate`. `per_page` is used as the activerecord/sql `limit`